workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java

```diff
@@ -108,33 +145,151 @@ public ResponseEntity<String> get(@PathVariable("issueKey") String issueKey)
 		path = "/jira/security-vulnerabilities/search"
 	)
 	public ResponseEntity<String> search(
-			@RequestParam(defaultValue = "", required = false) String keywords)
+			@RequestParam(defaultValue = "", required = false) String
+				filterAffectedVersions,
+			@RequestParam(defaultValue = "", required = false) String
+				filterCategories,
+			@RequestParam(defaultValue = "", required = false) String
+				filterClassifications,
+			@RequestParam(defaultValue = "", required = false) String
+				filterFixVersions,
+			@RequestParam(defaultValue = "", required = false) String
+				filterSeverities,
+			@RequestParam(defaultValue = "", required = false) String keywords,
+			@RequestParam(defaultValue = "", required = false) String page,
+			@RequestParam(defaultValue = "", required = false) String pageSize,
+			@RequestParam(defaultValue = "", required = false) String sortOrder)
 		throws Exception {
 
 		try {
-			StringBundler sb = new StringBundler(9);
+			StringBundler sb = new StringBundler(49);
```

Comment:
max `sb.append`s possible is 48 i think

Reply:
that number is updated by source formatter 😂

Reply:
I also counted 49 🤔